src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.ts

```diff
@@ -120,7 +120,7 @@ export class VehicleChecksCatCModal {
         map((vehicleChecks) => {
 
           // TODO: MES-4254 Use cat c Provider
-          return this.faultCountProvider.getVehicleChecksFaultCountCatBE(vehicleChecks);
+          return this.faultCountProvider.getVehicleChecksFaultCount(TestCategory.BE, vehicleChecks);
```

Comment:
why do you use `BE` here and `C` below? Is the comment still relevant here for future work or can it be done now?

Reply:
Ahmed is updating this component so his PR will change it :) Nice spot though